lib/ide/ast/impl/src/repr.rs

```diff
@@ -287,88 +253,58 @@ impl<T: HasRepr> HasRepr for TextBlockFmt<T> {
 
 // TODO: [mwu] `TextUnclosed<T>` as it needs to cut off closing quote from the
 //             stored text line. Likely this type should be stored like this.
-impl<T: HasSpan> HasSpan for TextUnclosed<T> {
-    fn span(&self) -> usize {
-        self.line.span() - 1 // remove missing quote
-    }
-}
 
-impl<T: HasRepr> HasRepr for TextUnclosed<T> {
-    fn write_repr(&self, target:&mut String) {
-        self.line.write_repr(target);
-        target.pop();  // remove missing quote
+// TODO: [jv] this implementation is wrong since we cannot `pop` from TokenBuilder
+//            either redesign TextUnclosed, so that it can use Tokenizer,
+//            or come up with some smart/ugly hack
```

Comment:
Please fix this before merge.
If smart solution doesn't come up quickly, just brute-force this and pattern match on the Line type, then emit proper opening quote and line segments.